custom_components/tuya_local/devices/royalclima_fresh_climate.yaml

```diff
@@ -370,4 +370,20 @@ secondary_entities:
     dps:
       - id: 103
         type: boolean
+  - entity: sensor
+    name: Power
+    class: power
+    category: diagnostic
+    dps:
+      - id: 116
+        type: integer
+        name: sensor
+        precision: 0
+        unit: W
+        class: measurement    
+        mapping:
+          - scale: 30
+        range:
+          min: 0
+          max: 1000000     
         name: sensor
```

Comment:
This name belongs to the sensor above, and needs to move back up there, otherwise the sensor above is missing a sensor dp, and this one has a duplicate name.